tests/unit/test_settings.py

```diff
@@ -28,6 +28,7 @@ def test_config__from_environ__ok():
             "GITHUB_TOKEN": "foo",
             "GITHUB_REPOSITORY": "owner/repo",
             "GITHUB_REF": "master",
+            "GITHUB_OUTPUT": "",
```

Comment:
I think the idea is to put as many non-default values here as possible and check that were properly transcribed below, so I'd go for a non-empty string.